lib/DBIx/TempDB.pm

```diff
@@ -85,7 +87,7 @@ use constant MAX_OPEN_FDS => eval { use POSIX qw( sysconf _SC_OPEN_MAX ); syscon
 
 our $VERSION = '0.06';
 
-our %SCHEMA_DATABASE = (postgresql => 'postgres', mysql => 'mysql');
+our %SCHEMA_DATABASE = (pg => 'postgres', mysql => 'mysql');
```

Comment:
The canonical_engine method from the URI::db will always return "pg" for any postgresql scheme.